backend/docker/monitoring/docker-compose.yml

```
@@ -228,3 +228,4 @@ volumes:
     driver: local
   tempo-logs:
     driver: local
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)